_posts/2017-06-08-Rust-1.18.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: "Announcing Rust 1.18"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.18.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed, getting Rust 1.18 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.18.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/rust-1.17-relnotes/RELEASES.md#version-1180-2017-06-08
+
+### What's in 1.18.0 stable
+
+Rust 1.18.0 is similar to many of our releases: no big bombshells, just a number
+of improvements, cleanups, and new features.
+
+One of the largest changes is a long time coming: core team members Carol
+Nichols and Steve Klabnik have been writing a new edition of "The Rust
+Programming Language", the official book about Rust. It's being [written openly
+on GitHub](https://github.com/rust-lang/book), and has over a hundred
+contributors in total. This release [includes the first draft of the second
+edition in our online documentation](https://doc.rust-lang.org/stable/book/).
+19 out of 20 chapters have a draft; the draft of chapter 20 will land in the
+Rust 1.19. When the book is done, a print version will be made available through
+[No Starch Press](https://www.nostarch.com/Rust), if you'd like a paper copy.
+While first drafts of the chapters are available, we're still working with the
+editors at No Starch to improve the text, so there's more to do, but we wanted
+to start getting a wider audience now. The new edition is a complete re-write
+from the ground up, using the last two years of knowledge we've gained from
+teaching people Rust. You'll find brand-new explanations for a lot of Rust's
+core concepts, new projects to build, and all kinds of other good stuff. Please
+check it out and let us know what you think!
```

Comment:
Should we include a link to the rust-lang/book repo again here? Or the new issue form in that repo?